packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_CoarseOperator_def.hpp

```diff
@@ -678,6 +1038,65 @@ namespace FROSch {
+            LO localVal = CoarseSolveMap_->getNodeNumElements();
+            LO sumVal;
+            LO minVal;
+            LO maxVal;
+
+            reduceAll(*CoarseSolveComm_,REDUCE_SUM,localVal,ptr(&sumVal));
+            SC avgVal = max(sumVal/double(CoarseSolveComm_->getSize()),0.0);
+            reduceAll(*CoarseSolveComm_,REDUCE_MIN,localVal,ptr(&minVal));
+            reduceAll(*CoarseSolveComm_,REDUCE_MAX,localVal,ptr(&maxVal));
+
+            if (CoarseSolveComm_->getRank() == 0) {
+                cout
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << setw(89) << "-----------------------------------------------------------------------------------------"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << "| "
+                << left << setw(74) << "Coarse problem statistics (coarse comm) " << right << setw(8) << "(Level " << setw(2) << this->LevelID_ << ")"
+                << " |"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << setw(89) << "========================================================================================="
+                // << "\n" << setw(FROSCH_INDENT) << " "
+                // << "| " << left << setw(41) << "Dimension of the coarse problem" << right
+                // << " | " << setw(41) << dimCoarseProblem
+                // << " |"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << "| " << left << setw(41) << "Number of ranks on the coarse comm" << right
+                << " | " << setw(41) << NumProcsCoarseSolve_
+                << " |"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << setw(89) << "-----------------------------------------------------------------------------------------"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << "| " << left << setw(20) << " " << right
+                << " | " << setw(10) << "total"
+                << " | " << setw(10) << "avg"
+                << " | " << setw(10) << "min"
+                << " | " << setw(10) << "max"
+                << " | " << setw(10) << "global sum"
+                << " |"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << setw(89) << "-----------------------------------------------------------------------------------------"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << "| " << left << setw(20) << "Number of rows" << right
+                << " | " << setw(10) << dimCoarseProblem
+                << " | " << setw(10) << avgVal
+                << " | " << setw(10) << minVal
+                << " | " << setw(10) << maxVal
+                << " | " << setw(10) << sumVal
+                << " |"
+                << "\n" << setw(FROSCH_INDENT) << " "
+                << setw(89) << "-----------------------------------------------------------------------------------------"
+                << endl;
+            }
+        }
+
```

Comment:
Why do we need this output a second time?

Reply:
I removed the second output.